metricbeat/module/coredns/stats/_meta/data_panic_event.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
We recently introduced support for `testdata`: https://github.com/elastic/beats/tree/master/metricbeat/module/kubernetes/apiserver/_meta/testdata What you need is a config.yml file inside this dir and then example .plain prometheus outputs.

You then can to got https://github.com/elastic/beats/tree/master/metricbeat/mb/testing and run `go test . -generate` and it will generate example outputs. You can also use a `docs.plain` which will generate the `data.json` output file for the docs. This will replace what you see in the expected files below.